sagemaker-builder/packing-a-tensorflow-model/packing_your_tensorflow_model.ipynb

```diff
@@ -0,0 +1,783 @@
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_OUTPUT_DIR=/opt/ml/output\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_INPUT_CONFIG_DIR=/opt/ml/input/config\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_OUTPUT_DATA_DIR=/opt/ml/output/data\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_INPUT_DATA_CONFIG={\"training\":{\"ContentType\":\"application/octet-stream\"}}\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_HOSTS=[\"algo-1-LQDWW\"]\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_NETWORK_INTERFACE_NAME=ethwe\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_MODEL_DIR=/opt/ml/model\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_INPUT_DIR=/opt/ml/input\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_MODULE_NAME=None\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_LOG_LEVEL=20\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_HPS={\"train-steps\":100}\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_CHANNELS=[\"training\"]\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_NUM_CPUS=2\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_CURRENT_HOST=algo-1-LQDWW\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_RESOURCE_CONFIG={\"current_host\":\"algo-1-LQDWW\",\"hosts\":[\"algo-1-LQDWW\"]}\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m SM_NUM_GPUS=0\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m \n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m Invoking script with the following command:\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m \n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m /usr/bin/python3 -m cifar10 --train-steps 100\n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m \n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m \n",
+      "\u001b[36malgo-1-LQDWW_1  |\u001b[0m 2018-10-27 17:21:38,784 sagemaker-containers INFO     Reporting training SUCCESS\n",
+      "\u001b[36mtmpnnll4yej_algo-1-LQDWW_1 exited with code 0\n",
+      "\u001b[0mAborting on container exit...\n",
+      "===== Job Complete =====\n"
+     ]
+    }
+   ],
+   "source": [
+    "from sagemaker.estimator import Estimator\n",
+    "\n",
+    "hyperparameters = {'train-steps': 100}\n",
+    "\n",
+    "instance_type = 'local'\n",
+    "\n",
+    "estimator = Estimator(role=role,\n",
+    "                      train_instance_count=1,\n",
+    "                      train_instance_type=instance_type,\n",
+    "                      image_name='tensorflow-cifar10-example:latest',\n",
+    "                      hyperparameters=hyperparameters)\n",
+    "\n",
+    "estimator.fit('file:///tmp/cifar-10-data')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 2: Training in Amazon SageMaker\n",
+    "Once you have your container pushed to ECR, you can use it to train models. Let's do that with the algorithm we made above."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Upload the data for training\n",
```

Comment:
I'd put the 'push the image' section first since preceding sentence identifies that as first task to do.